airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -139,12 +144,14 @@ protected void acceptTracked(AirbyteRecordMessage message) {
 
     // ignore other message types.
     final String streamName = message.getStream();
-    if (!streamNames.contains(streamName)) {
+    final AirbyteStreamNameNamespacePair pair = AirbyteStreamNameNamespacePair.fromRecordMessage(message);
+    LOGGER.info("pair: {}", pair);
```

Comment:
do we still know need this log line? it's going to log alot. once per record message.

Reply:
nope. good catch. removing!